app/screens/edit_profile/edit_profile.js

```diff
@@ -470,6 +470,7 @@ export default class EditProfile extends PureComponent {
                 theme={theme}
                 value={nickname}
                 isLandscape={isLandscape}
+                optional={true}
```

Comment:
The same should be applied to the position field

Reply:
ok!

Reply:
Please update the ticket too